modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -3563,6 +3563,14 @@
 		"issueKey": "LPD-47117",
 		"to": "addRole(null, param#0#, param#1#, param#2#, param#3#, param#4#, param#5#, param#6#, param#7#)"
 	},
+	{
+		"classNames": [
+			"PortletDisplay"
+		],
+		"from": "regex:(?<variableName>\\w+\\s*\\(?\\s*\\)?\\s*\\.\\s*)(?<prefixMethodName>getPortlet|setPortlet)Setup",
```

Comment:
@regiCesar21 could be simplified like this:
`"from": "regex:(?<variableName>\\w+[\w\s\(\)\.]*\\.)(?<prefixMethodName>getPortlet|setPortlet)Setup",`